repo/content/content_manager.go

```diff
@@ -453,7 +458,7 @@ func (bm *Manager) Flush(ctx context.Context) error {
 
 // RewriteContent causes reads and re-writes a given content using the most recent format.
 func (bm *Manager) RewriteContent(ctx context.Context, contentID ID) error {
-	log(ctx).Debugf("RewriteContent(%q)", contentID)
+	formatLog(ctx).Debugf("rewrite-content %v", contentID)
```

Comment:
parsing in tests sgtm, but will do it later, I want to get this change into 0.7.0 so that folks who run into issues will be able to share logs more easily.